server/sonar-server/src/test/java/org/sonar/server/issue/ws/ChangelogActionTest.java

```diff
@@ -291,4 +294,8 @@ private IssueDto newIssue() {
     return newDto(rule, file, project);
   }
 
+  private static String hash(String text) {
```

Comment:
we shouldn't duplicate the business logic. This is a good use case to fake the hash operation